03Lesson/01Homework/index.html

```diff
@@ -1,10 +1,34 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Berries</title>
   <link rel="stylesheet" href="css/style.css">
+  <meta charset="utf-8">
 </head>
 <body>
-
+    <main>
+      <section class="section1">
+        <div class="imageBox">
```

Comment:
The naming convention for CSS class names does not allow upper case. So this would have to be `image-box`, also in the css file then. 

Click on "Details" next to the failing CircleCI job, to see all errors. 
https://circleci.com/gh/AustinCodingAcademy/web-workbook/1029?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link